src/part02.rs

```diff
@@ -17,7 +17,7 @@ pub enum SomethingOrNothing<T>  {
 }
 // Instead of writing out all the variants, we can also just import them all at once.
 pub use self::SomethingOrNothing::*;
-//@ What this does is to define an entire family of types: We can now write
+//@ What this does is define an entire family of types: We can now write
```

Comment:
Yup I'm a native speaker.
In this case I'm certain that "is define" works best here.

If you wanted to use "is defining" you'd want to write something along the lines of "What we are doing is defining an entire family of types"